utility_analysis_new/parameter_tuning.py

```diff
@@ -0,0 +1,82 @@
+import pipeline_dp
+from dataclasses import dataclass
+import operator
+
+
+@dataclass
+class FrequencyBin:
+    """Represents 1 bin of the histogram.
+
+    The bin represents integers between 'lower' (inclusive) and 'upper'
+     (exclusive, not stored in this class).
```

Comment:
This is a bit ominous, maybe we could briefly mentiond that 'upper' is a deterministic function of 'lower', and we can mention what that function is.

Reply:
It makes sense about having deterministic function, added. Which function exactly, it's independent of the class itself, since there might be different conversions from n to bin